.settings/launch.json

```diff
@@ -5,7 +5,7 @@
 	"configurations": [
 		{
 			// Name of configuration; appears in the launch configuration drop down menu.
-			"name": "build",
+			"name": "livesync",

```

Comment:
:+1: , only basic setup of the file is required, which is already done, all other changes must be gitignored